git_machete/cmd.py

```diff
@@ -1320,53 +1320,53 @@ def __get_fork_point_override_data(self, b: str) -> Optional[Tuple[str, str]]:
+        self.__git.unset_config(self.config_key_for_override_fork_point_to(branch))
+        self.__git.unset_config(self.config_key_for_override_fork_point_while_descendant_of(branch))
 
-    def set_fork_point_override(self, b: str, to_revision: str) -> None:
-        if b not in self.__git.get_local_branches():
-            raise MacheteException(f"`{b}` is not a local branch")
+    def set_fork_point_override(self, branch: str, to_revision: str) -> None:
+        if branch not in self.__git.get_local_branches():
+            raise MacheteException(f"`{branch}` is not a local branch")
         to_sha = self.__git.get_commit_sha_by_revision(to_revision, prefix="")
         if not to_sha:
             raise MacheteException(f"Cannot find revision {to_revision}")
-        if not self.__git.is_ancestor_or_equal(to_sha, b, earlier_prefix=""):
+        if not self.__git.is_ancestor_or_equal(to_sha, branch, earlier_prefix=""):
             raise MacheteException(
-                f"Cannot override fork point: {self.__git.get_revision_repr(to_revision)} is not an ancestor of {b}")
+                f"Cannot override fork point: {self.__git.get_revision_repr(to_revision)} is not an ancestor of {branch}")
 
-        to_key = self.config_key_for_override_fork_point_to(b)
+        to_key = self.config_key_for_override_fork_point_to(branch)
         self.__git.set_config(to_key, to_sha)
 
-        while_descendant_of_key = self.config_key_for_override_fork_point_while_descendant_of(b)
-        b_sha = self.__git.get_commit_sha_by_revision(b, prefix="refs/heads/")
+        while_descendant_of_key = self.config_key_for_override_fork_point_while_descendant_of(branch)
+        b_sha = self.__git.get_commit_sha_by_revision(branch, prefix="refs/heads/")
         self.__git.set_config(while_descendant_of_key, b_sha)
 
         sys.stdout.write(
-            fmt(f"Fork point for <b>{b}</b> is overridden to <b>{self.__git.get_revision_repr(to_revision)}</b>.\n", f"This applies as long as {b} points to (or is descendant of) its current head (commit {self.__git.get_short_commit_sha_by_revision(b_sha)}).\n\n", f"This information is stored under git config keys:\n  * `{to_key}`\n  * `{while_descendant_of_key}`\n\n", f"To unset this override, use:\n  `git machete fork-point --unset-override {b}`\n"))
+            fmt(f"Fork point for <b>{branch}</b> is overridden to <b>{self.__git.get_revision_repr(to_revision)}</b>.\n", f"This applies as long as {branch} points to (or is descendant of) its current head (commit {self.__git.get_short_commit_sha_by_revision(b_sha)}).\n\n", f"This information is stored under git config keys:\n  * `{to_key}`\n  * `{while_descendant_of_key}`\n\n", f"To unset this override, use:\n  `git machete fork-point --unset-override {branch}`\n"))
 
-    def __pick_remote(self, b: str, is_called_from_traverse: bool) -> None:
+    def __pick_remote(self, branch: str, is_called_from_traverse: bool) -> None:
         rems = self.__git.remotes()
```

Comment:
While you're there, rename to `get_remotes` in the spirit of recent @garg-tanishka's change

Reply:
it fails when I change this.

Reply:
Hmmm what's the specific error? 🤔

Reply:
> While you're there, rename to `get_remotes` in the spirit of a recent @garg-tanishka's change

I think that's because `get_remoted()` doesn't exist. I think that was not part of her PR.

Reply:
Oh! right :) I've suggested to **rename** `remotes()` to `get_remotes()` on this very PR :)